ts/components/conversation/TimelineMessage.tsx

```diff
@@ -212,22 +213,36 @@ export function TimelineMessage(props: Props): JSX.Element {
         event.stopPropagation();
       }
 
-      if (!attachments || attachments.length !== 1) {
+      if (!attachments) {
         return;
       }
 
+      // check if any attachment needs to be downloaded from servers
       const attachment = attachments[0];
-      if (!isDownloaded(attachment)) {
-        kickOffAttachmentDownload({
-          attachment,
-          messageId: id,
-        });
-        return;
+      for (const attachment of attachments) {
+        if (!isDownloaded(attachment)) {
+          kickOffAttachmentDownload({
+            attachment,
+            messageId: id,
+          });
```

Comment:
We probably want a toast here telling the user why the save dialog isn't showing up.

Reply:
Yeah, probably pluralized for the single vs. multiple case.

Reply:
I added a new toast type and an action "showAttachmentDownloadStillInProgressToast" for this. 
Hope that was correct, but I didn't know how to show the toast otherwise.